tasks/system_probe.py

```diff
@@ -49,6 +49,7 @@ def build(
     windows=is_windows,
     arch="x64",
     embedded_path=DATADOG_AGENT_EMBEDDED_PATH,
+    compile_ebpf=True,
```

Comment:
I'm not seeing this flag used in this PR. Why was it added?

Reply:
Wrong rebase sorry. This flag is already on `main` and was part of an other PR. The reason of this flag is that sometimes recompiling the eBPF programs is not necessary - only modyfing the Go files - and it takes a fair amount of time.